src/Umbraco.Cms.Api.Management/Controllers/DataType/Items/ItemDatatypeItemController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Umbraco.Cms.Api.Management.ViewModels.DataType.Item;
+using Umbraco.Cms.Core.Mapping;
+using Umbraco.Cms.Core.Models;
+using Umbraco.Cms.Core.Services;
+
+namespace Umbraco.Cms.Api.Management.Controllers.DataType.Items;
+
+public class ItemDatatypeItemController : DatatypeItemControllerBase
+{
+    private readonly IDataTypeService _dataTypeService;
+    private readonly IUmbracoMapper _mapper;
+
+    public ItemDatatypeItemController(IDataTypeService dataTypeService, IUmbracoMapper mapper)
+    {
+        _dataTypeService = dataTypeService;
+        _mapper = mapper;
+    }
+
+    [HttpGet("item")]
+    [MapToApiVersion("1.0")]
+    [ProducesResponseType(typeof(IEnumerable<DataTypeItemResponseModel>), StatusCodes.Status200OK)]
+    public async Task<ActionResult> Item([FromQuery(Name = "key")] Guid[] keys)
```

Comment:
A general thing.. We should input a set, e.g. SortedSet<Guid>. so we dont have to handle duplicates. This needs to be changes on all endpoints